svencoop.asl

```diff
@@ -67,7 +81,18 @@ isLoading // Gametimer
 
 start // Start splitter
 {
-	if (settings["AutoStart"])
+	if (current.playerX >= -2092 && current.playerX <= -2004 && current.playerY >= 524 && current.playerY <= 720 && current.map == "uplink")
+		return true;
+
+	if (settings["Autostart"])
+	{
+		if (current.loading == 0 && old.loading == 1 && vars.startmaps.Contains(current.map))

```

Comment:
Does this only start the timer on "restart" command? Aka it wont "split", because its the "start" function?

Reply:
The timer will start only if you using **changelevel** cmd into console and this only works with those maps that are written in **vars.startmaps** (check 21 line).

This autostart does not respond to the **restart** or **map** cmd, so there will be no extra splits.

It's recommended to use in combination with autoreset, for example, if your run died on some map, you can just changelevel to start map and LiveSplit automatically reset all splits and timer too and start timer automatically.